libpromises/evalfunction.c

```diff
@@ -1586,6 +1623,44 @@ static FnCallResult FnCallGetMetaTags(EvalContext *ctx, ARG_UNUSED const Policy
         tagset = EvalContextClassTags(ctx, ref.ns, ref.name);
         ClassRefDestroy(ref);
     }
+    else if (strcmp(fp->name, "getbundlemetatags") == 0)    
+    {
+        const char *bundleref = RlistScalarValue(finalargs);
+        const Rlist *args = RlistFromSplitString(bundleref, ':');
+        const char *namespace;
+        const char *name;
+        if (!args->next)
+        {
+            namespace = "default";
+            name = RlistScalarValue(args);
+        } else {
+            namespace = RlistScalarValue(args);
+            name = RlistScalarValue(args->next);
+        }
+        const Bundle *bundle =  GetBundleFromPolicy(policy, namespace, name);
+        if (!bundle)
```

Comment:
```suggestion
        if (bundle == NULL)
```
Please use explicit comparisons as described in [CONTRIBUTING.md](https://github.com/cfengine/core/blob/master/CONTRIBUTING.md)